proposal.rst

```diff
@@ -168,27 +192,28 @@ Project Details
 
 **Describe any existing work and concepts on which your project is based.**
 
-FILLME
+My project is entirely based on the existing Xapian Bindings for Python and Swig's -builtin function.
 
 **Do you have any preliminary findings or results which suggest that your
 approach is possible and likely to succeed?**
 
-FILLME
+Xapian supports almost all of the languages other than PyPy. As Xapian has Python bindings, I'll also be able to write the same for PyPy.
+I tried using -builtin function and written few testcases to test the difference in speed and memory usage with and without -builtin. By modifying the typemaps for exception handling, we'll be able to use -builtin for Xapian.
 
 **What other approaches to have your considered, and why did you reject those in
 favour of your chosen approach?**
 
-FILLME
+The only part in my project which can have different approaches is for the trac issue #263. There are two approaches which Olly has suggested Either: doxy2swig.py should combine the comments for all the possible overloaded forms, or we should provide manually written overrides for all cases where these forms exist. I'll go with the former one as it will be much easier to combine the comments and show them.
```

Comment:
There's actually another potential option for this now - SWIG has fairly recently gained the ability to pick up doxygen-format documentation comments from the headers it is parsing and output them in the generated Python wrappers.

It's probably worth trying out this feature to see how it compares to `doxy2swig.py`.

Reply:
Okay, sure.